### ISSUE TITLE
Please add example of Play integration as standalone project

### ISSUE THREAD

**Issue by mkotsbak** [OlivierBlanvillain/scala-js-transport#4]
It is not obvious how to set up a Play project to use scalajs-transport (without Akka), and how to specify the dependencies. Please add a standalone example on how to do this.

**Comment by OlivierBlanvillain** [comment]
I think `examples/rpc` does just that (the build declaration is [here](https://github.com/OlivierBlanvillain/scalajs-transport/blob/master/build.sbt#L218), it does not depend on Akka). Do think I should provide an example using neither Autowire nor Akka? Or are you referring to the fact that the build file uses `dependsOn` instead of `libraryDependencies`?

**Comment by mkotsbak** [comment]
Yes, it has an internal dependency instead of to a Maven repo. But I found some hints in the buildfile of the game.
